exporting/init_connectors.c

```diff
@@ -37,6 +38,12 @@ int init_connectors(struct engine *engine)
                 if (init_opentsdb_http_instance(instance) != 0)
                     return 1;
                 break;
+            case BACKEND_TYPE_PROMETHEUS_REMOTE_WRITE:
+#if ENABLE_PROMETHEUS_REMOTE_WRITE
```

Comment:
Considering that when ` ENABLE_PROMETHEUS_REMOTE_WRITE` is not set we won't return 1 never, should you not put the `case` and `break` inside the `#if .. #endif`.

Reply:
I don't want it neither printing "EXPORTING: unknown exporting connector type" nor returning 1 in that case. The "Prometheus Remote Write support isn't compiled" message was already printed before the connectors initialization started.

Reply:
I got it, thanks!